cross/fuse/patch/cross-compile.patch

```diff
@@ -0,0 +1,25 @@
+--- util/Makefile.in.org    2015-08-19 10:56:51.834947550 +0200
```

Comment:
I missed this the first time around, but the dir must be called `patches`, not `patch`. Didn't you run into this during testing? If I'm reading spksrc.patch.mk correctly, the patch won't be applied otherwise.